client/post-editor/post-editor.jsx

```diff
@@ -231,9 +240,15 @@ export const PostEditor = React.createClass( {
 			<div className={ classes }>
 				<QueryPreferences />
 				<EditorConfirmationSidebar
-					hideSidebar={ this.hideConfirmationSidebar }
-					isActive={ this.state.showConfirmationSidebar }
+					closeOverlay={ this.closeConfirmationSidebarOverlay }
+					closeSidebar={ this.closeConfirmationSidebar }
+					isOverlayActive={ this.state.showConfirmationSidebarOverlay }
+					isSidebarActive={ this.state.showConfirmationSidebar }
```

Comment:
Ah, gotcha. The coupling feels a little odd. What about having a 'state' prop (name probably could be better) that could be set to one of: 'hidden', 'showConfirmation', 'publishing'? Then the sidebar can do whatever it needs to with the overlay and sidebar, but the editor doesn't need to be concerned with the details. Something to consider.